ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -716,40 +716,30 @@ public IXLTable InsertTable<T>(IEnumerable<T> data, String tableName, Boolean cr
 
             if (createTable)
                 // Create a table and save it in the file
-                return tableName == null ? range.CreateTable() : range.CreateTable(tableName);
+                return String.IsNullOrWhiteSpace(tableName) ? range.CreateTable() : range.CreateTable(tableName);
             else
                 // Create a table, but keep it in memory. Saved file will contain only "raw" data and column headers
-                return tableName == null ? range.AsTable() : range.AsTable(tableName);
+                return String.IsNullOrWhiteSpace(tableName) ? range.AsTable() : range.AsTable(tableName);
         }
 
         public IXLTable InsertTable(DataTable data)
         {
-            return InsertTable(data, null, true);
+            return InsertTable(data, true);
```

Comment:
In my opinion, it is better to change this to 
```
return InsertTable(data, data.TableName, true);
```

and not remove methods from the interface. This way, we won't introduce breaking changes, and not limit the functionality. Some users may want to dump a table to Excel with an arbitrary name not having to modify the `DataTable` instance as it may be used elsewhere. E.g. `TableName` matches the name in the database, while the Excel tables are built for different dates, or different customers, or whatever, and their names should be specified manually.

Reply:
@Pankraty good point.  I'll update your feedback.  Thx

Reply:
Add InsertTable methods back.  I still think IXLCell/XLCell could be simplified with the following:

```C#
// Currently
public IXLTable InsertTable(DataTable data)
public IXLTable InsertTable(DataTable data, Boolean createTable)
public IXLTable InsertTable(DataTable data, String tableName)
public IXLTable InsertTable(DataTable data, String tableName, Boolean createTable)

// To...
public IXLTable InsertTable(DataTable data, Boolean createTable = true)
public IXLTable InsertTable(DataTable data, String tableName, Boolean createTable = true)
```

But at least the other changes fix my original issue.